scraper/src/libretexts2zim/entrypoint.py

```diff
@@ -84,21 +94,41 @@ def main() -> None:
         required=True,
     )
 
+    parser.add_argument(
+        "--keep-cache",
+        help="Keep cache of website responses",
+        action="store_true",
+        default=False,
+    )
+
     args = parser.parse_args()
 
     logger.setLevel(level=logging.DEBUG if args.debug else logging.INFO)
 
+    output_folder = Path(args.output_folder)
+    output_folder.mkdir(exist_ok=True)
+    validate_zimfile_creatable(output_folder, "test.txt")
+
+    tmp_folder = Path(args.tmp_folder)
+    tmp_folder.mkdir(exist_ok=True)
+    validate_zimfile_creatable(tmp_folder, "test.txt")
```

Comment:
Something feels wrong about that func name 😀

Reply:
Indeed, thank you I forgot to open https://github.com/openzim/python-scraperlib/issues/200 (now it is done ^^)

Reply:
And I've tracked this as https://github.com/openzim/libretexts/issues/27